packages/core/getUserAvailability.ts

```diff
@@ -134,14 +136,15 @@ export async function getUserAvailability(
     eventTypeId,
     userId: currentUser.id,
     selectedCalendars,
+    beforeEventBuffer,
   });
 
   const bufferedBusyTimes: EventBusyDetails[] = busyTimes.map((a) => ({
     ...a,
-    start: dayjs(a.start).subtract(currentUser.bufferTime, "minute").toISOString(),
-    end: dayjs(a.end)
-      .add(currentUser.bufferTime + (afterEventBuffer || 0), "minute")
+    start: dayjs(a.start)
+      .subtract(afterEventBuffer || 0, "minutes")
```

Comment:
I think we need to add the beforeEventBuffer here too. I was able to book this with a beforeEventBuffer of 15 minutes:
<img width="215" alt="Screenshot 2022-10-28 at 15 33 52" src="https://user-images.githubusercontent.com/30310907/198613819-b7f34ec5-9c8d-46db-af95-ce8cb622197d.png">

Reply:
I will investigate this on Monday :) Although this is different behaviour from what myself and @Jaibles have experienced.

Reply:
Confirmed this is indeed a bug @CarinaWolli - nice spot. It happens when you book 2:30 before you book 2:15. beforeBuffer needs to be taken into account in the busyTime